applications/mlflow/kots/mlflow-chart.yaml

```diff
@@ -5,8 +5,10 @@ metadata:
 spec:
   chart:
     name: mlflow
-    chartVersion: 0.1.3
+    chartVersion: 0.2.0
   weight: 1
+  helmUpgradeFlags:
```

Comment:
should we also add `--timeout` flag to specify maximum wait time?

Reply:
yeah, i think that's a good idea. will do